test/smoke/src/areas/positron/top-action-bar/top-action-bar.test.ts

```diff
@@ -0,0 +1,108 @@
+				await app.workbench.quickaccess.openFile(join(app.workspacePathOrFolder, fileName2));
+				await app.workbench.quickaccess.runCommand('workbench.action.keepEditor', { keepOpen: false });
+				await app.workbench.editors.selectTab(fileName1);
+				await app.workbench.editor.waitForTypeInEditor(fileName1, text);
+				await app.workbench.editors.selectTab(fileName2);
+				await app.workbench.editor.waitForTypeInEditor(fileName2, text);
+				// The files are now "dirty" and the save buttons should be enabled
+				await app.workbench.editors.waitForTab(fileName1, true);
+				await app.workbench.editors.waitForTab(fileName2, true);
+				expect(await app.workbench.positronTopActionBar.saveButton.isEnabled()).toBeTruthy();
+				expect(await app.workbench.positronTopActionBar.saveAllButton.isEnabled()).toBeTruthy();
+				await app.workbench.positronTopActionBar.saveAllButton.click();
+				// The files are now saved, so the files should no longer be "dirty"
+				await app.workbench.editors.waitForTab(fileName1, false);
+				await app.workbench.editors.waitForTab(fileName2, false);
+				// The Save button stays enabled even when the active file is not "dirty"
+				expect(await app.workbench.positronTopActionBar.saveButton.isEnabled()).toBeTruthy();
+				// The Save All button is disabled when less than 2 files are "dirty"
+				expect(await app.workbench.positronTopActionBar.saveAllButton.isDisabled()).toBeTruthy();
+			});
+
+			it('Save and Save All both enabled when an unsaved new file is open', async function () {
+				const app = this.app as Application;
+				const fileName = 'Untitled-1';
+				const text = 'Bunnies hopping through a field of clover';
+				// Open a new file and type in some text
+				await app.workbench.quickaccess.runCommand('workbench.action.closeAllEditors', { keepOpen: false });
+				await app.workbench.quickaccess.runCommand('workbench.action.files.newUntitledFile', { keepOpen: false });
+				await app.workbench.editors.selectTab(fileName);
+				await app.workbench.editor.waitForTypeInEditor(fileName, text);
+				// The file is now "dirty" and the save buttons should be enabled
+				await app.workbench.editors.waitForTab(fileName, true);
+				expect(await app.workbench.positronTopActionBar.saveButton.isEnabled()).toBeTruthy();
+				expect(await app.workbench.positronTopActionBar.saveAllButton.isEnabled()).toBeTruthy();
+				await app.workbench.positronTopActionBar.saveButton.click();
+				// The file is now saved, so the file should no longer be "dirty"
+				await app.workbench.editors.waitForTab(fileName, false);
+				// The Save button stays enabled even when the active file is not "dirty"
+				expect(await app.workbench.positronTopActionBar.saveButton.isEnabled()).toBeTruthy();
+				// The Save All button is disabled when less than 2 files are "dirty"
+				expect(await app.workbench.positronTopActionBar.saveAllButton.isDisabled()).toBeTruthy();
```

Comment:
Not sure why but for me locally and for the run in CI, the above line seems like it should be toBeFalsy.  Thoughts?